src/mmw/js/src/draw/utils.js

```diff
@@ -84,15 +85,39 @@ function cancelDrawing(map) {
     map.fire('draw:drawstop');
 }
 
-// Return shape area in km2.
-function shapeArea(shape) {
```

Comment:
I can imagine `shapeArea` being useful enough to keep around, but I suppose we can always bring it back when we need it.